source/NanostackRfPhyAT86RF215.cpp

```diff
@@ -1183,13 +1193,26 @@ void RFBits::rf_irq_task(void)
     }
 }
 
-int RFBits::init_215_driver(RFBits *_rf, TestPins *_test_pins, const uint8_t mac[8], uint8_t *rf_part_num)
+int RFBits::init_215_driver(RFBits *_rf, TestPins *_test_pins, Se2435Pins *_se2435_pa_pins, const uint8_t mac[8], uint8_t *rf_part_num)
```

Comment:
Is it possible to have this as different setter function to enable the PA so you don't need to change the init interface? I think it would be easier to add support different PAs in future

Reply:
Sure, I'll change it